pkg/resources/etcd/statefulset.go

```diff
@@ -17,9 +17,12 @@ limitations under the License.
 package etcd
 
 import (
+	"bytes"
 	"fmt"
 	"strconv"
+	"text/template"
 
+	"github.com/Masterminds/sprig"
```

Comment:
Please separate 3rd party non-k8s imports with an empty line from the kubermatic packages.